### PR TITLE
adding KB links to java

### PR DESCRIPTION
**Pull request by l0k0ms**

### CODE REVIEW

File: content/integrations/java.md [hunk 1]
```diff
@@ -336,8 +336,10 @@ instances:
   5. Restart the agent: `sudo /etc/init.d/datadog-agent`
 
 
+## Further Reading
 
-
-
-
-
+* [I Have a Matching Bean for my JMX integration but nothing on Collect !](https://help.datadoghq.com/hc/en-us/articles/218277463-I-Have-a-Matching-Bean-for-my-JMX-integration-but-nothing-on-Collect-)
```
Comment (jeremy-lq) [inline content/integrations/java.md]: Is the extraneous space between Collect and the ! intentional?

File: content/integrations/java.md [hunk 2]
```diff
@@ -336,8 +336,10 @@ instances:
...
+* [View jmx data in jConsole and set up your jmx.yaml to collect them](https://help.datadoghq.com/hc/en-us/articles/207525586-View-jmx-data-in-jConsole-and-set-up-your-jmx-yaml-to-collect-them)
```
Comment (jeremy-lq) [inline content/integrations/java.md]: I like consistency, so I'd say s/jmx/JMX/ as it's capitalized everywhere else.

File: content/integrations/java.md (hunk 1 above)
Comment (l0k0ms) [reply content/integrations/java.md]: it to have more impact :) reviewed 👍

File: content/integrations/java.md (hunk 2 above)
Comment (l0k0ms) [reply content/integrations/java.md]: 👍

**Pull request merged**